docs/reference/indices/seal.asciidoc

```diff
@@ -1,21 +1,32 @@
 [[indices-seal]]
 == Seal
 
-The seal API allows to flush one or more indices and adds a marker to primaries and replicas if there are no pending write operations.
-The seal marker is used during recovery after a node restarts. If a replica is allocated on a node which already has a shard copy with the same seal as the primary then no files will be copied during recovery.
-Sealing is a best effort operation. If write operations are ongoing while the sealing is in progress then writing the seal might fail on some copies.
+The seal API flushes and adds a "seal" marker to one or more indices. The seal
+is used during recovery or restarts to skip the first and most costly phase of
+the process if all copies of the shard have the same seal. No segment files
+need to be copied and the transaction log replay phase of the recovery can
+start immediately which should make recovery much, much faster.
 
-A seal marks a point in time snapshot (a low level lucene commit). This mark can be used to decide if the initial, rather resource heavy, recovery phase where segments or event the entire lucene index is copied over the network can
-be skipped. If the indices on both sides of the recover have the same seal no segment files need to be copied and transaction log replay can start immediately. The seal breaks as soon as the shard issues a new lucene commit, uncommitted operations
-in the transaction log do not break the seal until they are committed.
+There are two important points about seals:
+1. They are best effort in that if there are any outstanding write operations
+while the seal operation is being performed then the shards which those writes
+target won't be sealed but all others will be. See below for more.
+2. The seal breaks as soon as the shard issues a new lucene commit. Uncommitted
+operations in the transaction log do not break the seal. That is because a seal
+marks a point in time snapshot of the segments, a low level lucene commit.
+Practically that means that the simplest way to not break the seal is to not
+modify the content of the index and that once the seal is broken its useless.
 
```

Comment:
Yeah, you are right, it will only be removed eventually. Maybe "That means that every write operation will cause the seal to be removed eventually"? Not sure how to formulate this non-passively :)